src/main/resources/tasks/init/model/16.ts

```diff
@@ -0,0 +1,63 @@
+import type {WriteConnection} from '/lib/explorer/types/index.d';
+import type {SynonymNodeSpecific} from './16/modifySynonymNodeEditor';
+
+
+import {toStr} from '@enonic/js-utils';
+import {setModel} from '/lib/explorer/model/setModel';
+import {Progress} from '../Progress';
+import {getSynonymsForThesaurusName} from './16/getSynonymsForThesaurusName';
+import {getThesauri} from './16/getThesauri';
+import {modifySynonymNodeEditor} from './16/modifySynonymNodeEditor';
+
+
+export function model16({
+	progress,
+	writeConnection
+} :{
+	progress :Progress
+	writeConnection :WriteConnection
+}) {
+	progress.addItems(1).setInfo('Get all thesauri to lookup language:{from , to}...').report().logInfo();
+	const thesauri = getThesauri(writeConnection);
+	log.debug('thesauri:%s', toStr(thesauri));
```

Comment:
Remove

Reply:
Not yet, I want to look at it when I create the migration